extension/js/background_page/background_page.htm

```diff
@@ -1,21 +0,0 @@
-<html>
-  <head>
-    <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
-  </head>
-
-  <body>
-    <script src="/lib/trusted-types.js"></script>
-    <script src="/lib/purify.js"></script>
-    <script src="/lib/jquery.min.js"></script>
-    <script src="/lib/openpgp.js"></script>
```

Comment:
@sosnovsky  As we removed background_page, now opgp becomes null for below code.
I'm thinking about those 3 solutions.

1. Import ES module in service worker (background.ts). Maybe it's openpgp.min.mjs?. We can't just import openpgp.js (by use `import lib/openpgp.js` as openpgp.js code is `something like var openpgp = (function (exports) {....})()`)
2. Use webpack or other solution to combine libraries like openpgp in backgroun.ts
3. Use `[importScripts](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts)` to load library (AFAIK, it is deprecated)?

What's your take on this?
Please let me know your suggestion.

https://github.com/FlowCrypt/flowcrypt-browser/blob/4edbf1836cff75b6262bacc1757f45382661520c/extension/js/common/core/crypto/pgp/openpgp-key.ts#L39

Reply:
> Use [importScripts](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts) to load library (AFAIK, it is deprecated)?

I didn't find any information about it's deprecation, and it's recommended at https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/basics, so we can try to use `importScripts` for importing `openpgp`

Reply:
I found it from stackoverflow I think but couldn't find link.
Here's alternative link
https://www.mediawiki.org/wiki/Topic:Ufathwjhyszsis17

Reply:
It seems this wiki article is specific for mediawiki, while for chrome extensions `importScripts` is a recommended solution for importing js files, let's try to use it

Reply:
Okay, Got it

Reply:
You know we are using servie worker as a module therefore we can't use `importScripts`. If we remove `module`, then `import` statements won't work in `background.ts` file

> Uncaught TypeError: Failed to execute 'importScripts' on 'WorkerGlobalScope': Module scripts don't support importScripts().

Therefore there are 2 options left from my suggestion?

> - Import ES module in service worker (background.ts). Maybe it's openpgp.min.mjs?. We can't just import openpgp.js (by use import lib/openpgp.js as openpgp.js code is something like var openpgp = (function (exports) {....})())
> - Use webpack or other solution to combine libraries like openpgp in backgroun.ts
> 

https://github.com/FlowCrypt/flowcrypt-browser/blob/056d979c3704e60ca05b4a04adf9b5aaa6dc3e73/extension/manifest.json#L70

Reply:
Another solution I'm thinking is make our custom `importScripts` function which fetches library first and then execute it with `eval` or something like that. But it might come with security risk

Reply:
Yeah, it's better to use some ready solutions than write custom functions, which can be broken after some browser update.

So let's try to import ES module in service worker, and see if it works for our case.